packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+import inquirer from "inquirer";
+import semver from "semver";
+import { readFileSync, writeFileSync } from 'fs';
+import { logger } from "../utils/logger.js";
+import { execSync } from "child_process";
+
+export async function updateCommand() {
+    try {
+        const installedPackages = JSON.parse(execSync('npm list --depth=0 --json').toString());
+        const triggerDevPackages = Object.keys(installedPackages.dependencies).filter(pkg => pkg.startsWith('axios'));
```

Comment:
I think this should probably be `pkg.startsWith("@trigger.dev/")`, right?

Reply:
fixed